### PR TITLE
replace-c-style-format-with-f-strings

### PR DESCRIPTION
**Pull request by nicksonlangat**

### CODE REVIEW

File: indigo/bulk_creator.py [hunk 1]
```diff
@@ -1223,7 +1223,7 @@ def find_work(self, given_string):
             # but it could be in the current batch
             for row in self.works:
                 if hasattr(row, 'work') and (substring == row.work.frbr_uri or given_string == row.work.title):
-                    work = _('%s (about to be imported)') % given_string
```
Comment (longhotsummer) [inline indigo/bulk_creator.py:1226]: this has to remain as-is, because the interpolation has to happen after the translation (the `_(...)`).

**Review by longhotsummer — CHANGES_REQUESTED** [review]
Hi @nicksonlangat thanks for this, and sorry for the delay. It looks good, just the one with the translation which should be undone.

**Comment by nicksonlangat** [comment]
> Hi @nicksonlangat thanks for this, and sorry for the delay. It looks good, just the one with the translation which should be undone.

Updated pr

**Comment by longhotsummer** [comment]
Thanks. Holding off until https://github.com/laws-africa/indigo/pull/1827 is merged which should fix the tests.

**Comment by longhotsummer** [comment]
@nicksonlangat pls merge master into your branch, that should fix the tests.

**Comment by nicksonlangat** [comment]
> @nicksonlangat pls merge master into your branch, that should fix the tests.

Done

**Comment by longhotsummer** [comment]
I'm closing because this is no so out of date :(

**Pull request closed without merging**